src/rpc/masternode.cpp

```diff
@@ -93,44 +93,75 @@ UniValue privatesend(const JSONRPCRequest& request)
 
 UniValue getpoolinfo(const JSONRPCRequest& request)
 {
-    if (request.fHelp || request.params.size() != 0)
-        throw std::runtime_error(
-            "getpoolinfo\n"
-            "Returns an object containing mixing pool related information.\n");
+    throw std::runtime_error(
+        "getpoolinfo\n"
+        "DEPRECATED. Please use getprivatesendinfo instead.\n"
+    );
+}
 
-#ifdef ENABLE_WALLET
-    CPrivateSendBaseManager* pprivateSendBaseManager = fMasternodeMode ? (CPrivateSendBaseManager*)&privateSendServer : (CPrivateSendBaseManager*)&privateSendClient;
+UniValue getprivatesendinfo(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() != 0) {
+        throw std::runtime_error(
+            "getprivatesendinfo\n"
+            "Returns an object containing an information about PrivateSend settings and state.\n"
+            "\nResult (for regular nodes):\n"
+            "{\n"
+            "  \"enabled\": true|false,             (bool) Whether mixing functionality is enabled\n"
+            "  \"running\": true|false,             (bool) Whether mixing is currently running\n"
+            "  \"multisession\": true|false,        (bool) Whether PrivateSend Multisession option is enabled\n"
+            "  \"max_sessions\": xxx,               (numeric) How many parallel mixing sessions can there be at once\n"
+            "  \"max_rounds\": xxx,                 (numeric) How many rounds to mix\n"
+            "  \"max_amount\": xxx,                 (numeric) How many " + CURRENCY_UNIT + " to keep anonimized\n"
```

Comment:
@UdjinM6 Just found this one. 
s/anonimized/anonymized/

Reply:
🙈 
smth to fix together with other trivial fixes I guess :)